src/infinite-hits/infinite-hits.ts

```diff
@@ -12,6 +12,22 @@ import { BaseWidget } from '../base-widget';
 import { NgAisInstantSearch } from '../instantsearch/instantsearch';
 import { noop } from '../utils';
 
+export type InfiniteHitsState = {
+  hits: InfiniteHitsItem[];
+  results: {};
+  isFirstPage: boolean;
+  isLastPage: boolean;
+  showMore: Function;
+  showPrevious: Function;
+};
+
+export type InfiniteHitsItem = {
```

Comment:
The [`Hit` type](https://github.com/algolia/instantsearch.js/blob/65911504b9ae0e2596c75f0ae74133985dc79850/src/types/instantsearch.ts#L46-L71) won't be subject to change anytime soon because it comes straight from the [Algolia API response](https://www.algolia.com/doc/api-reference/api-methods/search/#method-response-hits) (except `__position` and `__queryID` which are internals).

I hear your argument but I think that it doesn't apply for this type. Sure, it'd be great if we had an easy way to share them across projects already.

Reply:
ok. I've updated the type in a common place so that it can be shared between Hits and InfiniteHits. The PR is here https://github.com/algolia/angular-instantsearch/pull/579.

As for this PR, I'll just set InfiniteHitsItem to Hit.

Reply:
done here: https://github.com/algolia/angular-instantsearch/pull/553/commits/bdd8f082aff9ca3fd0d71dbf7bb575db0ec37dd8